docs/blueprint-designer-guide/service-accounts-for-aws.md

```diff
@@ -12,13 +12,13 @@ The basic process is as follows:
 
 ## Prerequisites
 
-*	IAM OIDC provider on the cluster’s account, to recognize the cluster on the account
-*	kubectl on the cluster
-*	AWS CLI on your computer
+*	[IAM OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) on the cluster’s account, to recognize the cluster on the account
+*	[kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) on the cluster
```

Comment:
I don't think it's on the cluster I think it's also on the local computer just needs to be connected to the relevant cluster